frontend/apps/pwa/src/features/browser/RecordField.tsx

```diff
@@ -0,0 +1,42 @@
+import { FieldDefinition, Record } from 'core-api-client';
+import { FC } from 'react';
+import { Link } from 'react-router-dom';
+import format from 'date-fns/format';
+
+type RecordFieldProps = {
+  field: FieldDefinition;
+  value: any;
+  subRecords: Record[] | undefined;
+};
+export const RecordField: FC<RecordFieldProps> = (props) => {
+  const { field, value, subRecords } = props;
+
+  function renderField(f: FieldDefinition, v: any) {
```

Comment:
I think using arrow functions are typically better as they have block scoping, clearer currying syntax, and align with modern js practices better.

Reply:
completely agree, copy paste issues ...